### PR TITLE
Add files via upload

### PR DESCRIPTION
**Pull request by Faribah**
Please review and comment on the SQL scripts  and advice on creating a combo Primary key with the first two columns for viva_insights_meeting table. More details included in the email sent last week.

### CODE REVIEW

**Review by wung-ms — COMMENTED** [review]
There is a mention in the use case in the readme on combining data. Would the schema support this?

**Comment by Faribah** [comment]
Yes the schema has a secondary employeeId (e.g. zid in the wpa.ai tenant) for this purpose.

**Comment by cwarren-msft** [comment]
CREATE TABLE [dbo].[zCoryTest_delme2]
(
[MeetingId] [nvarchar](4000) NOT NULL,
[StartTimestampUTC] [datetime2](7) NOT NULL,
[EndTimestampUTC] [datetime2](7) NOT NULL,
[Organizer_PersonId] [nvarchar](4000) NOT NULL,
[Organizer_Organization] [nvarchar](4000) NULL,
[Organizer_LevelDesignation] [nvarchar](4000) NULL,
[Organizer_IsInternal] [bit] NULL,
[Attendees] [int] NULL,
**_PRIMARY KEY ([MeetingId],[StartTimestampUTC])_**
)


Need a better way to do composite primary key
Result:

Warning! The maximum key length for a clustered index is 900 bytes. The index 'PK__zCoryTes__4918EBB662A27C48' has maximum length of 8008 bytes. For some combination of large values, the insert/update operation will fail.

File: Sample Solutions/Viva Insights Query Analysis Automation/SQLscripts/MeetingQueryTbl.sql [hunk 1]
```diff
@@ -0,0 +1,43 @@
+SET ANSI_NULLS ON
+GO
+SET QUOTED_IDENTIFIER ON
+GO
+
+CREATE TABLE [dbo].[viva_insights_meeting]
+( 
+	[MeetingId] [nvarchar](4000)  NOT NULL,
```
Comment (cwarren-msft) [inline Sample Solutions/Viva Insights Query Analysis Automation/SQLscripts/MeetingQueryTbl.sql:8]: CREATE TABLE [dbo].[zCoryTest_delme2]
(
[MeetingId] nvarchar NOT NULL,
[StartTimestampUTC] datetime2 NOT NULL,
[EndTimestampUTC] datetime2 NOT NULL,
[Organizer_PersonId] nvarchar NOT NULL,
[Organizer_Organization] nvarchar NULL,
[Organizer_LevelDesignation] nvarchar NULL,
[Organizer_IsInternal] [bit] NULL,
[Attendees] [int] NULL,
**### PRIMARY KEY ([MeetingId],[StartTimestampUTC])**
)

Need a better way to do composite primary key
Result:

Warning! The maximum key length for a clustered index is 900 bytes. The index 'PK__zCoryTes__4918EBB662A27C48' has maximum length of 8008 bytes. For some combination of large values, the insert/update operation will fail.

**Pull request closed without merging**